pyirf/irf/background.py

```diff
@@ -56,3 +56,71 @@ def background_2d(events, reco_energy_bins, fov_offset_bins, t_obs):
     bg_rate = per_energy / t_obs / bin_solid_angle
 
     return bg_rate.to(BACKGROUND_UNIT)
+
+
+def background_3d(events, reco_energy_bins, fov_offset_bins, t_obs):
+    """
+    Calculate background rates in square bins in the field of view.
+
+    GADF documentation here:
+    https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/bkg/index.html#bkg-3d
+
+    Parameters
+    ----------
+    events: astropy.table.QTable
+        DL2 events table of the selected background events.
+        Needed columns for this function: `reco_fov_lon`, `reco_fov_lat`,
+        `reco_energy`, `weight`.
+    reco_energy: astropy.units.Quantity[energy]
+        The bins in reconstructed energy to be used for the IRF
+    fov_offset_bins: astropy.units.Quantity[angle]
```

Comment:
`fov_offset` is the name we use for the radial distance from the center of the field of view.

We should use two arguments here, `fov_lon_bins` and `fov_lat_bins`